webextensions/api/runtime.json

```diff
@@ -1180,13 +1180,15 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/runtime/setUninstallURL",
             "support": {
               "chrome": {
-                "version_added": "41"
+                "version_added": "41",
+                "notes": "To Chrome 114 supported a maximum length of 255 characters, from Chrome 115 supports a maximum length of 1023."
```

Comment:
This doesn't read well at a first glance. How about something like

> Since Chrome 115, the maximum length is 1023. Until Chrome 114, the maximum length was 255.

Or the following (my favorite because it's clear and short):

> The maximum length was increased from 255 to 1023 in Chrome 115.

This applies to all other notes in this PR.